### PR TITLE
fix: restore appauth

### PR DESCRIPTION
**Pull request by tibco-tdinh**
# Story

Restores "@openid/appauth" library to package.json

## Changes

1. @openid/appauth

## Tests

Manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,6 @@
     "build": "ng build on-prem",
     "build-cloud": "ng build cloud",
     "build-common": "ng build common",
-    "build-all": "ng build common --configuration production && ng build cloud --configuration production  && ng build on-prem --configuration production",
```
Comment (Gerromie) [inline package.json:16]: Why was this removed? I found it helpful in preparing to build the docker containers.

File: package.json [hunk 2]
```diff
@@ -45,8 +45,6 @@
     "chartjs-plugin-doughnutlabel": "^2.0.3",
     "chartjs-plugin-labels": "^1.1.0",
     "chartjs-plugin-streaming": "^2.0.0",
-    "luxon": "^2.3.0",
-    "heatmap.js": "git+https://github.com/sanuradhag/heatmap.js.git",
```
Comment (Gerromie) [inline package.json:49]: These need to be present or the tests will fail, along with probably some functionality.

Comment (tibco-tdinh) [reply package.json:49]: I've added them back. For whatever reason, I didn't see them in the merge conflicts or merge changes.
